src/content/docs/en/guides/migrate-to-astro/from-nextjs.mdx

```diff
@@ -2,73 +2,906 @@
+      params: { name },
+    }))
+
+  return {
+  paths: pokemons.map(({ name }) => ({
+    params: { name },
+  }))
+}
+
+const { name } = Astro.props;
+---
+
+<Layout>
+  <img class={styles.pokeImage} src={pokemon.image} alt={`${pokemon.name} picture`} />
+  <div class={styles.infoContainer}>
+    <h1 class={styles.header}>No. {pokemon.id}: {pokemon.name}</h1>
+    <table class={styles.pokeInfo}>
+      <tbody>
+        <tr>
+          <th>Types</th>
+          <td>{pokemon.types}</td>
+        </tr>
+        <tr>
+          <th>Height</th>
+          <td>{pokemon.height}</td>
+        </tr>
+        <tr>
+          <th>Weight</th>
+          <td>{pokemon.weight}</td>
+        </tr>
+      </tbody>
+    </table>
+    <p class={styles.flavor}>{pokemon.flavorText}</p>
+  </div>
+</Layout>
+```
+
+2. Remove the `getStaticProps()` method wrapping the API data fetch, as in the previous example. Add this, along with all other necessary JavaScript for the templating, to the Astro page code fence.
+
+
+
+```astro ins={16-33} title="src/pages/pokemon/[name].astro"
+---
+import styles from '../../styles/pokemon-entry.module.css';
+
+export const getStaticPaths = async () => {
+  const res = await fetch("https://pokeapi.co/api/v2/pokemon?limit=151")
+  const resJson = await res.json();
+  const pokemons = resJson.results;
+
+  return pokemons.map(({ name }) => ({
+      params: { name },
+    }))
+}
+
+const { name } = Astro.props;
+
+function capitalize(str) {
+  return str.charAt(0).toUpperCase() + str.slice(1);
+}
+
+const [pokemonData, species] = await Promise.all([
+    fetch(`https://pokeapi.co/api/v2/pokemon/${name}`).then(res => res.json()),
+    fetch(`https://pokeapi.co/api/v2/pokemon-species/${name}`).then(res => res.json())
+])
+
+const pokemon = {
+    id: pokemonData.id,
+    image: pokemonData.sprites.front_default,
+    name: capitalize(pokemonData.name),
+    height: pokemonData.height,
+    weight: pokemonData.weight,
+    flavorText: species.flavor_text_entries[0].flavor_text,
+    types: pokemonData.types.map(({ type }) => type.name).join(', ')
+};
+---
+
+<Layout>
+  <img class={styles.pokeImage} src={pokemon.image} alt={`${pokemon.name} picture`} />
+  <div class={styles.infoContainer}>
+    <h1 class={styles.header}>No. {pokemon.id}: {pokemon.name}</h1>
+    <table class={styles.pokeInfo}>
+      <tbody>
+        <tr>
+          <th>Types</th>
+          <td>{pokemon.types}</td>
+        </tr>
+        <tr>
+          <th>Height</th>
+          <td>{pokemon.height}</td>
+        </tr>
+        <tr>
+          <th>Weight</th>
+          <td>{pokemon.weight}</td>
+        </tr>
+      </tbody>
+    </table>
+    <p class={styles.flavor}>{pokemon.flavorText}</p>
+  </div>
+</Layout>
+``` 
+
+*/}
 
 ## Community Resources 
 
+- Video: [NextJS to Astro: more control = faster sites](https://www.youtube.com/watch?v=PSzCtdM20Fc).
+
+- Video: [How Astro made my site 100x faster](https://www.youtube.com/watch?v=cOxA3kMYtkM).
+
```

Comment:
- Blog post: [Migrating from Next.js to Astro](https://johnzanussi.com/posts/nextjs-to-astro-migration).